app/Console/Commands/BulkReviewPosts.php

```diff
@@ -60,13 +60,15 @@ public function handle()
         $logfreq = $this->option('logfreq');
         $log = $this->option('log');
         $tags = $this->option('tag') ?? null;
-
+        info('rogue:bulkreviewposts: Beginning to query');
         $posts = Post::setEagerLoads([])
             ->where('campaign_id', $this->argument('campaign'))
             ->where('status', $this->argument('oldStatus'))
             ->where('type', $this->argument('type'))
             ->get();
 
+        info('rogue:bulkreviewposts: Posts result from query: ' . $posts);
```

Comment:
let's just log the first result `->first()`

Reply:
also can you limit the query to 10 posts (instead of all of them) has any issues?

Reply:
ah yes forgot that! Updated :)